frontend/components/organiser/event/details/EventDetailsEdit.tsx

```
@@ -783,59 +782,57 @@ export const EventDetailsEdit = ({
+                    </Select>
+                  </div>
+                ) : (
+                  <div className="mt-2">
+                    {attachForm === null ? (
+                      "No form attached"
+                    ) : (
+                      <div
+                        className="flex items-center gap-2 w-fit px-1 py-0.5 -mx-1 -my-0.5 rounded cursor-pointer hover:bg-gray-100 transition-colors duration-200"
+                        onClick={() => router.push(`/organiser/forms/${attachForm.formId}/preview`)}
+                        title="View form preview"
                       >
```

Biome (2.4.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

(lint/a11y/useKeyWithClickEvents)